src/openforms/forms/tests/test_migrations.py

```diff
@@ -435,3 +435,57 @@ def test_fix_badly_formatted_rules(self):
             self.rule_badly_formatted.json_logic_trigger,
             {"==": [{"var": "selectBoxes.a"}, True]},
         )
+
+
+class TestLogicParsing(TestMigrations):
+    migrate_from = "0002_auto_20210917_1114_squashed_0045_remove_formstep_optional"
+    migrate_to = "0046_convert_advanced_logic"
+    app = "forms"
+
+    def setUpBeforeMigration(self, apps):
+        self.FormDefinition = apps.get_model("forms", "FormDefinition")
```

Comment:
no reason to assign the model to `self` here - ideally you want the new version of the model in the test anyway